api/src/http.rs

```diff
@@ -400,18 +400,33 @@ impl Default for ProxyConfig {
 
 /// Configuration for mirror.
 #[derive(Clone, Deserialize, Serialize, Debug)]
+#[serde(default)]
 pub struct MirrorConfig {
     /// Mirror server URL, for example http://127.0.0.1:65001.
     pub host: String,
     /// HTTP request headers to be passed to mirror server.
-    #[serde(default)]
```

Comment:
why removing this? Are `headers` and `auth_through` mandatory now?

Reply:
No, headers and auth_through is not mandatory.
We have implemented Default trait for MirrorConfig, so remove this. 😊
Are this right?

Reply:
`#[serde(default)]` is for serde, that is to handle the case when this field is missed from json string. It serves different purpose of `#[derive(Default)]`

Reply:
Got it. I will fix this error. Thank you very much.

Reply:
Done.